src/main/resources/public/jsx/geo-dash-widget-editor.js

```diff
@@ -825,30 +844,30 @@ class BasicLayout extends React.PureComponent{
         let ed = new Date(this.state.endDate);
         let sd = new Date(this.state.startDate);
         if(! this.state.dualLayer) {
-            if (ed > sd && this.state.FormReady != true) {
-                this.setState({FormReady: true})
+            if (ed > sd && this.state.FormReady !== true) {
```

Comment:
looks like the same logic as above.  make this a helper function that returns true/false